import "MsgHead.proto";    


message MsgTechnologyStateReq
{

}

message MsgOpenTechLevelReq
{
	required int32 level = 1;
}

message MsgApplyAddTechInfoReq
{
	required int32 level		= 1;
	required int32 id			= 2;
	required int32 addcount		= 3;
}

message MsgAddTechValueReq
{
	required int32 count		= 1;
}